todo.js

```diff
@@ -0,0 +1,130 @@
+
+//DEFINITIONS
+var bullet = "- ";
+
+var button = $("<button>").attr("type", "submit").text("submit");
+var input = $("<input>").attr("type", "string");
+var label = $("<label>").append("Add an item <br>").append(this.input);
+
+var icon = function(src) {
```

Comment:
When I made the Icon object constructor (? not sure about terminology), I was considering features like "edit item" button that may be implemented in the future, hence the src parameter.